api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -292,6 +292,23 @@
 	@Authorized(PrivilegeConstants.EDIT_ORDERS)
 	public Order updateOrderFulfillerStatus(Order order, Order.FulfillerStatus orderFulfillerStatus, String fullFillerComment);
 	
+	/**
+	 * Updates the fulfillerStatus of an order and the related comment and finally persists it
+	 *
+	 * @param order order whose fulfillerStatus should be changed
+	 * @param orderFulfillerStatus describes the new Order.FulfillerStatus the order should be set to
+	 * @param fullFillerComment is a string which describes a comment that is set while changing the FulfillerStatus 
+	 * @param accessionNumber is the accession number of the order             
```

Comment:
should be "is the accession number to set"

Reply:
it done